src/btree/bt_vrfy.c

```diff
@@ -78,6 +82,11 @@ __verify_config(WT_SESSION_IMPL *session, const char *cfg[], WT_VSTUFF *vs)
     WT_RET(__wt_config_gets(session, cfg, "dump_pages", &cval));
     vs->dump_pages = cval.val != 0;
 
+    WT_RET(__wt_config_gets(session, cfg, "read_corrupt", &cval));
+    vs->read_corrupt = cval.val != 0;
+    vs->verify_err = 0;
+    vs->corrupt_pages = 0;
```

Comment:
AFAICT this field is not used for anything. There is no statistic or message or other use for this other than counting. If it isn't needed or used, it should be removed. I think the user will get N messages out from reading in however many bad pages in other code.